mrmoody-metrics/README.md

```diff
@@ -0,0 +1,61 @@
+### Metrics for Mr.Moody
+
+### Install Go
+https://golang.org/doc/install
```

Comment:
So, the only way to run it is to install go on host machine ? Or is there an alternate way of running this using docker ?